app/templates/admin/content/events.hbs

```diff
@@ -57,31 +59,33 @@
           </div>
         </div>
         <div class="ui divider"></div>
-        <table class="ui celled table">
-          <tbody>
-            {{#each model.eventTopics as |eventTopic|}}
-              <tr>
-                <td>
-                  <div class="ui grid">
-                    <div class="middle aligned content eight wide column left aligned">
-                      {{eventTopic.name}}
-                    </div>
-                    <div class="eight wide column right aligned">
-                      <div class="ui horizontal compact basic buttons">
-                        {{#ui-popup content=(t 'Edit') class='ui icon button' click=(action 'openModalFor' eventTopic)}}
-                          <i class="edit icon"></i>
-                        {{/ui-popup}}
-                        {{#ui-popup content=(t 'Delete') class='ui icon button' click=(action (confirm (t 'Are you sure you would like to delete this event topic?') (action 'deleteEventProperty' eventTopic)))}}
-                          <i class="trash outline icon"></i>
-                        {{/ui-popup}}
+        <div style = "max-height: 768px;overflow-x: hidden;">
```

Comment:
We strictly prohibit the use of custom styling or css in `hbs`, You will rarely see any custom css in the web-app. Please try to use `semantic ui classes`

Reply:
@kushthedude @maze-runnar This is fine for hotfix but I have one doubt, would it not mess up with mobile views?

Reply:
i think, it would not -
![Screenshot from 2019-11-12 19-14-10](https://user-images.githubusercontent.com/56407566/71325143-a5d21580-250e-11ea-92a9-51a8bdca2abf.png)
![Screenshot from 2019-11-12 19-17-21](https://user-images.githubusercontent.com/56407566/71325144-a5d21580-250e-11ea-81d8-317c3b8c589e.png)


> @kushthedude @maze-runnar This is fine for hotfix but I have one doubt, would it not mess up with mobile views?